actors/builtin/miner/miner_actor.go

```diff
@@ -255,6 +259,16 @@ func (a Actor) SubmitWindowedPoSt(rt Runtime, params *SubmitWindowedPoStParams)
 	if params.Deadline >= WPoStPeriodDeadlines {
 		rt.Abortf(exitcode.ErrIllegalArgument, "invalid deadline %d of %d", params.Deadline, WPoStPeriodDeadlines)
 	}
+	if params.ChainCommitEpoch >= currEpoch {
+		rt.Abortf(exitcode.ErrIllegalArgument, "PoSt chain commitment %d must be in the past", params.ChainCommitEpoch)
+	}
+	if params.ChainCommitEpoch < currEpoch-MaxPoStChainCommitAge {
```

Comment:
What happens for epochs <= MaxPoStChainCommitAge?

Reply:
I don't think I understand the question. The chain commit must be sufficiently recent, or the proof will be rejected.

Reply:
I mean, what happens when the current epoch is less than MaxPoStChainCommitAge? I.e., less than a day old. It looks like this will wrap around and "just work", but that seems funky.

Reply:
Ah, nevermind. I keep forgetting that we allow negative chain epochs.